### PR TITLE
 Fix shader variant stripping in cloud build.

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 Fixes issues that was preventing the shader stripper to run in cloud build. This caused builds using CloudBuild and batchmode to have insane amount of times and shader data. 
 An option to choose loggin shader variants was added to the asset.
---
### Release Notes
 Added

---
### Testing status
**Katana Tests**: 

**Manual Tests**: Tested manually.

**Automated Tests**: No automated tests required.

Any test projects to go with this to help reviewers? Tested with the LWRP Template.

---
### Overall Product Risks
**Technical Risk**: None. 
**Halo Effect**: None.

---
### Comments to reviewers
 I also took the chance to remove that static `supportedShaderFeatures`. The stripper now figures which variants to strip based on the current assigned pipeline asset.
@Katstodian please review not only the release notes but also the tooltips/text in the Asset Editor file.

### CODE REVIEW

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added occlusion mesh rendering/hookup for VR
 - You can now configure default depth and normal shadow bias values in the pipeline asset.
 - You can now add the `LWRPAdditionalLightData` component to a `Light` to override the default depth and normal shadow bias.
+- You can now selected to log the amount of shader variants in the build by selecting `Shader Variant Log Level` under `Advanced` group in the pipeline asset.
+### Changed
+- Removed `supportedShaderFeatures` property from LWRP core. The shader stripper now figures which variants to strip based on the current assigned pipeline asset in graphics settings.
```
Comment (Katstodian) [inline com.unity.render-pipelines.lightweight/CHANGELOG.md]: Please replace with: 
Removed the `supportedShaderFeatures` property from LWRP core. The shader stripper now figures out which variants to strip based on the current assigned pipeline Asset in the Graphics settings.

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ### Fixed
+- Previously projects were taking an insane amount of time to build when building from Cloud Build. Now, shader variant stripping is working when building a project using CloudBuild.
```
Comment (Katstodian) [inline com.unity.render-pipelines.lightweight/CHANGELOG.md]: Please replace with:
Shader variant stripping now works when you're building a Project with Cloud Build. This greatly reduces build times from Cloud Build.

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 3]
```diff
@@ -9,7 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added occlusion mesh rendering/hookup for VR
 - You can now configure default depth and normal shadow bias values in the pipeline asset.
 - You can now add the `LWRPAdditionalLightData` component to a `Light` to override the default depth and normal shadow bias.
+- You can now selected to log the amount of shader variants in the build by selecting `Shader Variant Log Level` under `Advanced` group in the pipeline asset.
```
Comment (Katstodian) [inline com.unity.render-pipelines.lightweight/CHANGELOG.md]: Please replace with:
You can now log the amount of shader variants in your build. To do so, go to the Pipeline Asset, select __Advanced__, and select __Shader Variant Log Level__. Here, you can select __Disabled__, __Lighweight RP only__, or __All shaders__

Comment (phi-lira) [reply com.unity.render-pipelines.lightweight/CHANGELOG.md]: IMO this is too much information for a changelog. Can we omit the "Here, you can select Disabled, Lighweight RP only, or All shaders". This should be in docs instead. Changelogs should just raise awareness of things to people and bugfixes. Details should be in the docs.

**Pull request merged**